frontend/src/__tests__/cypress/cypress/utils/s3Buckets.ts

```diff
@@ -0,0 +1,3 @@
+import type { AWSS3Bucket } from '~/__tests__/cypress/cypress/types';
+
+export const AWS_PIPELINES_BUCKET: AWSS3Bucket = Cypress.env('AWS_PIPELINES_BUCKET');
```

Comment:
In our Jenkins CI we use `S3.BUCKET_2` for the pipeline bucket (according to our test-variables.yaml). 
Would this also work if calling it in bash: 
`export AWS_PIPELINES_BUCKET=S3.BUCKET_2`

Reply:
@FedeAlonso if that is not working well, then you can add the required bucket details to our test-variables downstream (But for the long term, we should try to use the same variable names if they already exist in test-variables. Thanks!

Reply:
Fixed in 
28d8d14